src/main/java/mat/server/CQLConstantServiceImpl.java

```diff
@@ -75,6 +78,12 @@
 
 	private static final long serialVersionUID = 1L;
 
+
+	@Value("${mat.fhirMatServices.url}")
+	private String fhirMatServicesUrl;
+
+	private final String fhirMatServiceResouceForAttributes = "/find";
```

Comment:
JSL naming convention for final fields is upper case

Reply:
Done